wasp/apps/alarm.py

```diff
@@ -0,0 +1,155 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2020 Daniel Thompson
+# Copyright (C) 2020 Joris Warmbier
+"""Alarm Application
+~~~~~~~~~~~~~~~~~~~~
+
+An application to set a vibration alarm. All settings can be accessed from the Watch UI
```

Comment:
Please add a screenshot to the header comment. This gets things ready for the app to go into the [Application Library](https://wasp-os.readthedocs.io/en/latest/apps.html). You can use the 's' button in the simulator to generate the screenshot and check ouf the Game of Life app ([wasp/apps/gameoflife.py](https://github.com/daniel-thompson/wasp-os/blob/master/wasp/apps/gameoflife.py) ) for an example of how to link to a screenshot from the header text.